### DISCUSSION TITLE
A new icon is needed

### DISCUSSION THREAD

**Discussion by gunaygultekin** [tailwindlabs/heroicons#917] [Ideas]
Hi, thanks for the current icons. They are very usefull.
However,  I couldnt find a parallel icon, like below. Please could you add ?  Thanks

<img width="144" alt="image" src="https://user-images.githubusercontent.com/15944706/213677211-3fed69f5-65e1-4456-8cbf-11026f81b039.png">

<img width="209" alt="image" src="https://user-images.githubusercontent.com/15944706/213677276-a9133bbb-3c54-4e4e-8f41-f54529116e83.png">

**Comment by zefir-git** [comment]
Looks a lot like `fork` icon. Other git icons like `merge` would also be really nice.
